azurerm/azurerm_container_group/simple/main.tf

```diff
@@ -0,0 +1,49 @@
+# Summary: A simple Azure Container Group
+
+# Documentation: https://www.terraform.io/docs/language/settings/index.html
+terraform {
+  required_version = ">= 0.14.0"
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "~> 2.0"
+    }
+  }
+}
+
+# Documentation: https://www.terraform.io/docs/language/values/variables.html
+variable "azure_subscription_id" {
+  type = string
+}
+
+# Documentation: https://www.terraform.io/docs/language/providers/requirements.html
+# Documentation: https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs
+provider "azurerm" {
+  features {}
+
+  subscription_id = var.azure_subscription_id
+}
+
+# Resource Group
+# Documentation: https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/resource_group
+resource "azurerm_resource_group" "changeme_simple_container_group_resource_group" {
+  name     = "changeme-simple-kubernetes-cluster-resource-group"
+  location = "West Europe"
+}
+
+# Container Group within the Resource Group
+# Documentation: https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/container_group
+resource "azurerm_container_group" "changeme_simple_container_group" {
+  name                = "changeme-simple-container-group"
+  location            = azurerm_resource_group.changeme_simple_container_group_resource_group.location
+  resource_group_name = azurerm_resource_group.changeme_simple_container_group_resource_group.name
+
+  os_type = "Linux"
+
+  container {
+    name   = "hello-world"
+    image  = "microsoft/aci-helloworld:latest"
+    cpu    = "0.5"
+    memory = "1.5"
+  }
+}
```

Comment:
It gives this error from terraform apply:
* https://github.com/ContainerSolutions/terraform-examples/runs/3263126672?check_suite_focus=true#step:5:1120

```
azurerm_container_group.changeme_simple_container_group: Creating...
╷
│ Error: Error creating/updating container group "changeme-simple-container-group" (Resource Group "changeme-simple-kubernetes-cluster-resource-group"): containerinstance.ContainerGroupsClient#CreateOrUpdate: Failure sending request: StatusCode=0 -- Original Error: Code="MissingIpAddressPorts" Message="The ports in the 'ipAddress' of container group 'changeme-simple-container-group' cannot be empty."
│ 
│   with azurerm_container_group.changeme_simple_container_group,
│   on main.tf line 36, in resource "azurerm_container_group" "changeme_simple_container_group":
│   36: resource "azurerm_container_group" "changeme_simple_container_group" ***
│ 
╵
/home/runner/work/terraform-examples/terraform-examples/azurerm/azurerm_container_group/simple
```

Reply:
There's an open issue in terraform-providers/terraform-provider-azurerm for this: https://github.com/terraform-providers/terraform-provider-azurerm/issues/1697 

```
The documentation for azurerm_container_group resource describes optional ip_address_type and container port arguments without default values
```

Reply:
Added a fix for this:
* https://github.com/ContainerSolutions/terraform-examples/pull/73/commits/f24ae495ee57f3467d9a1ecc339f6913e0ac6e38